static/src/javascripts/bootstraps/common.js

```diff
@@ -251,6 +255,28 @@ define([
                 new Clickstream({filter: ['a', 'button']});
             },
 
+            showAdblockMessage: function () {
+                var alreadyVisted = storage.local.get('alreadyVisited') || 0,
```

Comment:
What's the already visited thing for here? `Message` already has something where it remembers when it's been closed in local storage and doesn't appear again. It does it by the id (`adblock` here).